### ISSUE TITLE
binary outcomes

### ISSUE THREAD

**Issue by macartan** [DeclareDesign/DesignLibrary#158]
Suggest the canonical designers should have binary outcome option using probit type dgp
Documentation change to reflect outcome_means etc are defined on the latent space if binary invoked
Alernative is to make separate designers.

**Comment by jaspercooper** [comment]
OK I think for many designers this may not be too hard. 

Since we're mostly working in a standard normal space, one approach is simply to do a probit-style transformation of the POs. 

That's my preferred approach but we would need to warn (either in function or documentation) that in such cases parameters such as ICC and rho pertain to the latent variable and not the realized Y. 

Another more laborious approach would be to have an entirely separate DGP. That would entail either an ifelse logic or an rlang implementation in every one of the designers, which for me has the drawback of making source code hard to read for people that want to write their own designers.

**Comment by jaspercooper** [comment]
I'm hesitant to create parallel designers because I worry about slippage

**Comment by macartan** [comment]
SGTM

would it be possible to simply allow and function f that transforms Xb;
default is identifiy but people cuolld do logits, probits exponentials,
create counts or whatever, at tehri own risk

**Comment by nfultz** [comment]
Agreed it would better to support a link function - I don't think estimatr
has any functionality for that though.
